.travis/build_release_archive.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mv -f .gitattributes.release .gitattributes
+git add .
```

Comment:
The add & commit is used only to make use of `git archive` feature.
The committed changes are never pushed to the repository.